pkg/api/plugin.go

```diff
@@ -14,10 +14,22 @@ const (
 	ContextKeyTenantID     ContextKey = "TenantID"
 )
 
+// TestConfig is embedded into PluginConfig
+type TestConfig struct {
```

Comment:
nits - can you place this below `type PluginConfig struct` for readability and add a comment on PluginConfig that TestConfig should not be populated in production.

Reply:
The last part of this comment is still outstanding.